lib/document/stem.js

```diff
@@ -1,5 +1,8 @@
 const fileUrl = require('file-url')
-const mathjaxFileUrl = fileUrl(require.resolve('mathjax/es5/tex-chtml-full.js'))
+const ospath = require('path')
+
+let mathjaxFileUrl = process.pkg ? ospath.join(process.execPath, '..', 'assets/mathjax/tex-chtml-full.js') :
```

Comment:
```suggestion
const mathjaxFileUrl = process.pkg ? ospath.join(process.execPath, '..', 'assets/mathjax/tex-chtml-full.js') :
```

Reply:
Here we do not check that `process.pkg` is not undefined using `typeof process.pkg !== 'undefined'`. Any particular reason?

Reply:
I shall make it consistent. No particular reason except bad me.

Reply:
I borrowed this code from: https://github.com/rocklau/pkg-puppeteer/issues/2 I think given this is from someone more experienced with puppeteer, we'll keep the form `const isPkg = typeof process.pkg !== 'undefined';`